bin/cli.js

```diff
@@ -0,0 +1,234 @@
+  [ '-p, --browser-port <n>',    'Browser port' ],
+  [ '-c, --config <value>',      'Config file' ],
+];
+
+const karmaLogLevels = {
+  'disable': [ 'LOG_DISABLE', 'none' ],
+  'error':   [ 'LOG_ERROR', 'errors-only' ],
+  'warn':    [ 'LOG_WARN', 'minimal' ],
+  'info':    [ 'LOG_INFO', 'normal' ],
+  'debug':   [ 'LOG_DEBUG', 'verbose' ],
+};
+
+const browserLaunchers = {
+  'Chrome':         'karma-chrome-launcher',
+  'ChromeHeadless': 'karma-chrome-launcher',
+  'Firefox':        'karma-firefox-launcher',
+  'IE':             'karma-ie-launcher',
+  'Opera':          'karma-opera-launcher',
+  'Safari':         'karma-safari-launcher',
+};
+
+const logLevels = ['quiet', 'disable', 'error', 'warn', 'info', 'debug'];
+
+const isLogAtLeast = (level1, level2) => {
+  const value1 = logLevels.indexOf(level1);
+  const value2 = logLevels.indexOf(level2);
+  if (value1 === -1) level1 = 'disable';
+  if (value2 === -1) level2 = 'disable';
+  return value1 >= value2;
+};
+
+const merge = (arr1 = [], arr2 = []) => common.merge(arr1, arr2);
+
+const exclude = (files, filters) => filters.map(path => path.replace('.', '\\.')
+  .replace('*', '.+')
+  .replace('?', '.'))
+  .map(path => new RegExp(path))
+  .reduce((files, regexp) => files.filter(file => !regexp.test(file)), files);
+
+const parseFile = file => {
+  const data = fs.readFileSync(path.resolve(file), 'utf8');
+  switch (common.fileExt(file)) {
+    case 'json':
+      return JSON.parse(data);
+    case 'yml':
+    case 'yaml':
+      return yaml.parse(data);
+    default:
+      throw new Error('Unsupported config file format');
+  }
+};
+
+const loadFiles = files => {
+  const result = [];
+  files.map(file => {
+    if (fs.existsSync(file + '.js')) {
+      return file + '.js';
+    } else if (fs.existsSync(file)) {
+      return file;
+    } else {
+      console.error('File does not exist:', file);
+      process.exit(1);
+      return '';
+    }
+  }).forEach(file => {
+    if (fs.statSync(file).isDirectory()) {
+      const subfiles = fs.readdirSync(file).map(f => path.join(file, f));
+      result.push(...loadFiles(subfiles));
+    } else if (common.fileExt(file) === 'js') {
+      result.push(file);
+    }
+  });
+  return result;
+};
+
+const setKarmaBrowsers = (config, ...browsers) => {
+  if (!config.plugins) config.plugins = [];
+  if (!config.browsers) config.browsers = [];
+
+  browsers.forEach(browser => {
+    const launcher = browserLaunchers[browser];
+    if (!launcher) {
+      console.error(
+        `Metatests library does not support such browser: ${browser}`
+      );
+      process.exit(1);
+    }
+    config.browsers.push(browser);
+    if (!config.plugins.includes(launcher)) config.plugins.push(launcher);
+  });
+};
+
+const setKarmaLogLevel = (config, logLevel = 'disable') => {
+  if (logLevel === 'quiet') logLevel = 'disable';
+  if (!config.webpackMiddleware) config.webpackMiddleware = {};
+  config.logLevel = karma.constants[karmaLogLevels[logLevel][0]];
```

Comment:
Why would we need to change `config.logLevel` here?
Even if we do need to change it, why isn't it `config.logLevel = logLevel`?

Reply:
Of course, it can be changed like this
```js
for(level in karmaLogLevels) {
  karmaLogLevels[level][0] = karma.constants[karmaLogLevels[level][0]];
}

...

config.logLevel = karmaLogLevels[logLevel][0];
```
WDYT?

Reply:
> Having either karma or webpack values as our internal not only bind us strictly to their values but also removes choice to add more levels or combine log levels of karma and webpack.

That's exactly what I'm asking, why do you bind the value from `karma.constants` to `config.logLevel` if you pass `logLevel = 'disable'` (log level in our terms)?

Reply:
This config will be passed to `karma` and its log level depends on `config.log` which must be the value of one of the constants mentioned above. If we want to be able to change our `logLevel` names and what they will do we have to convert our `logLevel` to `logLevel` for `karma` and `webpackMiddleware.stats` for `webpack`.

If you are talking about `quiet`, then yes, it makes sense to skip setting log levels.

`karma-log-level` as an argument would be incorrect because we also have `webpack` which has its own log level independent from log level of `karma`.